lib/l10n/intl_messages.arb

```diff
@@ -3101,5 +3101,65 @@
     "type": "text",
     "placeholders_order": [],
     "placeholders": {}
+  },
+  "camera": "Camera",
+  "@camera": {
+    "type": "text",
+    "placeholders_order": [],
+    "placeholders": {}
+  },
+  "audio_recording": "Audio recording",
+  "@audio_recording": {
+    "type": "text",
+    "placeholders_order": [],
+    "placeholders": {}
+  },
+  "give_access_in_settings": "Please give linshare access in the settings to continue",
+  "@give_access_in_settings": {
+    "type": "text",
+    "placeholders_order": [],
+    "placeholders": {}
+  },
+  "permission_required": "Permission required",
+  "@permission_required": {
+    "type": "text",
+    "placeholders_order": [],
+    "placeholders": {}
+  },
+  "recording_saved": "Recording saved",
+  "@recording_saved": {
+    "type": "text",
+    "placeholders_order": [],
+    "placeholders": {}
+  },
+  "permission_denied": "Permission denied",
+  "@permission_denied": {
+    "type": "text",
+    "placeholders_order": [],
+    "placeholders": {}
+  },
+  "error_while_recording": "An error has occured while recording",
+  "@error_while_recording": {
+    "type": "text",
+    "placeholders_order": [],
+    "placeholders": {}
+  },
+  "you_did_not_record_anything": "You did not record anything",
+  "@you_did_not_record_anything": {
+    "type": "text",
+    "placeholders_order": [],
+    "placeholders": {}
+  },
+  "exit": "exit",
```

Comment:
Discard

Reply:
@KhaledNjim please dont forget it

Reply:
What do i discard ?

Reply:
I mean, change it to `Discard` or `Exit`.